### DISCUSSION TITLE
Is Leather Engine going to be more optimized in the future?

### DISCUSSION THREAD

**Discussion by ghost** [Leather128/LeathersFunkinEngine#260] [Q&A] [answered]
I kind of have a pretty bad pc right now, and I don't expect a new one for at least a year.
This is a problem to me by Leather Engine since I would need basically every optimization there would be, including the Characters and Backgrounds option being turned off.
I can't see shit besides arrows in songs, which sucks.
I'm just wondering if you are able to make Leather Engine more optimized in the future if you can.

**Comment by Leather128** [comment, marked as answer]
uh i'm not really sure how to optimize it more without rewriting a ton of code, also those optimization options *specifically* turn off actual game elements like backgrounds and stuff (that's what the option does) so yeah
